music21/environment.py

```diff
@@ -432,6 +432,18 @@ def _loadDefaults(self, forcePlatform=None):
             ]:
                 self.__setitem__(name, value)  # use for key checking
 
+    def _checkAccessibility(self, path: Union[str, pathlib.Path]) -> bool:
```

Comment:
path typing should include 'None' as a possibility, since rootTempDir can be None, and otherwise the isinstance check does not make sense below.

Reply:
@tanchihpin0517 this was my fault, sorry, you just committed my suggestion.

```suggestion
    def _checkAccessibility(self, path: Optional[Union[str, pathlib.Path]]) -> bool:
```